draft.xml

```diff
@@ -365,6 +365,11 @@ mfa_token=eyJhbGciOiJ...&challenge_type=otp%20oob
                 REQUIRED. The type of the MFA challenge issued.  Value is case
                 insensitive.
               </t>
+
+              <t hangText='authenticator_id'>
```

Comment:
A different option might be adding the details an ui will need (name and oob_channel) directly in the a response instead of an id, I think that option is a little bit clearer because it does not assumes that there is a method to list authenticators / get details of one authenticator (that belong to a separate RFC). Having said that I'm not particularly in pro of one or the other.